web3/contract.py

```diff
@@ -778,14 +778,22 @@ def call_contract_function(contract,
     except DecodingError as e:
         # Provide a more helpful error message than the one provided by
         # eth-abi-utils
-        msg = (
-            "Could not decode contract function call {} return data {} for "
-            "output_types {}".format(
-                function_name,
-                return_data,
-                output_types
+        empty_strings = ["0x", b"0x", ""]
```

Comment:
Can you move this to the top level of the module so it doesn't have to be re-allocated everytime.  Also, rename to cap-cased and something bytecode specific like `EMPTY_BYTECODE_VALUES`